docs/user_guide/logging.rst

```diff
@@ -0,0 +1,57 @@
+Logging
+=======
+
+Since version 0.18.0 :program:`rebase-helper` uses standard module-level loggers.
+:program:`rebase-helper` uses :class:`rebasehelper.logger.CustomLogger` logger class which provides extra
+logging levels.
+
+Default console handlers can be setup by calling :meth:`rebasehelper.logger.create_stream_handlers` and default file
+handlers by calling :meth:`rebasehelper.logger.create_file_handlers`.
+
+There are also 3 loggers with a special purpose:
+
+=============================== =====================================================================
+Logger name                     Purpose
+=============================== =====================================================================
+:samp:`rebasehelper.traceback`  traceback generated by unhandled exceptions
+:samp:`rebasehelper.summary`    messages forming the rebase summary
+:samp:`rebasehelper.report`     messages forming the textual rebase report (used by text output tool)
+=============================== =====================================================================
+
+
+Examples
+--------
+
+.. code-block:: python
+    :caption: myapp.py
+
+    import logging
+
+    from rebasehelper.specfile import SpecFile
+
+    # Add a handler, alternatively, rebasehelper.logger.create_stream_handlers
+    # could be used to setup the default handlers.
+    logger = logging.getLogger('rebasehelper.specfile')
+    handler = logging.StreamHandler()
+    handler.setLevel(logging.DEBUG)
```

Comment:
I'd rather use [logging.basicConfig()](https://docs.python.org/3/library/logging.html#logging.basicConfig), to keep the example as simple as possible.